### ISSUE TITLE
Find a cheaper basic CRM system

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1478]
We currently pay about £70/month for a CRM system that handles creating tickets for enquiries to hello@ or organisers@. It basically ensures that all the organisers can see and respond to all the messages that come in.

When we just had it forwarding, it was too easy for people to forget to reply-all, and then for someone to be off for a few days and miss important emails.

We don't need anything fancy, but we do need the ability to make sure everyone gets all replies (although if we don't find anything simple soon it's not worth continuing to pay £70).

**Comment by MatthewCroughan** [comment]
From your description, I do not see why this could not be used in a similar to way as you might use Jira to create tickets. It is essentially an open source trello, no doubt there is an API you can use to create issues on the board which could act like tickets. https://github.com/plankanban/planka

**Comment by johnmckerrell** [comment]
For the record, the emails/conversations would need to be private but if this tool would work with a private repo then that might work.

**Comment by MatthewCroughan** [comment]
Yeah, it's entirely self hosted so there are no limitations on privacy. When I say issues, I do not mean github issues, I just mean the general concept of issues. This tool would not interface with github in any way as far as I know.

**Comment by johnmckerrell** [comment]
TBH I'm really wondering if it's possible to do it with exim linked to a database. Needs something along the lines of:

1. email arrives at hello@does - this could simply be forwarded to newticket@support.does...
2. ticket is created with some sort of ID
3, a set of email addresses are assigned to ticket, as well as all email addresses on the original email (To & CC)
4, email contents are sent to everybody assigned to ticket who has not already received it, with a reply-to of <ticketid>@support.does...

Then further replies are handled with:

1. Email comes in to <ticketid>@support.does...
2. Essentially GOTO 3 above but we'd only add any people who weren't already assigned

One troublesome thing would be requiring some sort of unsubscribe mechanism, which might require a website, although I guess you could perhaps do it by email.

Also you wouldn't want ticket IDs to be guessable. You could perhaps just ignore emails to a ticketid if the person wasn't already connected to the ticket but often an existing person will want to add someone in. You could then allow this specific behaviour, but it's too easy to spoof email addresses so you could just add yourself to any ticket by spoofing an email from an existing organiser.

Would also have to consider spam too, and would likely need to send emails out through AWS's SMTP servers to make sure our email wasn't flagged as spam.

**Comment by johnmckerrell** [comment]
I've just gone through the list on this page: https://alternative.me/supportbee

Zendesk at $5/agent/month is reasonably cheap.
Zoho has 3 agents free then additional £10/month each https://www.zoho.com/desk/pricing.html

Few self hosted options:
 * https://bestpractical.com/request-tracker
 * https://osticket.com/

**Comment by JessWade** [comment]
[Techloyce.com](https://www.techloyce.com) also providing the same services with very reasonable pricing. Their certified Zoho Consultants provide specialist consulting knowledge in Zoho applications, allowing you to improve business efficiency and get the most out of your technology investments by utilizing the Zoho app suite. By engaging with you and understanding your business requirements, we can help you match your company goals with the newest technology and streamline your business operations with Zoho automation. Our consultation service also aids in the success of CRM systems at all levels.

**Comment by amcewen** [comment]
I haven't dug into them, but there's been a similar [discussion on the CoTech forum](https://community.coops.tech/t/what-software-do-you-use-for-handling-support-requests/3039/2) which seems to like Zammad
